src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBodyMapper.java

```diff
@@ -199,4 +199,22 @@ private CCDAddress getDefendantAddress(CCDRespondent respondent) {
             ? respondent.getPartyDetail().getPrimaryAddress()
             : respondent.getClaimantProvidedDetail().getPrimaryAddress();
     }
+
+    public DocAssemblyTemplateBody paperResponseAdmissionLetter(CCDCase ccdCase, String caseworkerName) {
+        LocalDate currentDate = LocalDate.now(clock.withZone(UTC_ZONE));
+        String partyName = ccdCase.getRespondents().get(0)
+            .getValue().getPartyName() != null
+            ? ccdCase.getRespondents().get(0).getValue().getPartyName() :
+            ccdCase.getRespondents().get(0).getValue().getClaimantProvidedPartyName();
```

Comment:
A bit easier to read if you set the defendant to a variable
```suggestion
        CCDRespondent defendant =  ccdCase.getRespondents().get(0).getValue();
        String partyName = defendant.getPartyName() != null
            ? defendant.getPartyName() 
            :  defendant.getClaimantProvidedPartyName();
```